web/src/message_list_tooltips.ts

```diff
@@ -379,20 +394,28 @@ export function initialize(): void {
             assert(message_lists.current !== undefined);
             const message_container = message_lists.current.view.message_containers.get(message_id);
             assert(message_container !== undefined);
-            if (message_container.last_edit_timestamp === undefined) {
+            if (
+                message_container.last_edit_timestamp === undefined &&
+                message_container.last_moved_timestamp === undefined
+            ) {
                 return false;
```

Comment:
This would not be new in this PR, but I wonder if `local_edit_timestamp` should be handled here. Probably a good #frontend follow-up conversation.

Reply:
When we set `message_container.moved` in the message list view module, we set `message_container.last_edit_timestamp` by checking first for `local_edit_message`. So it's already been handled.

Reply:
Added some code comment documentation to the refactor commit for these tooltips, so that my reasoning was a bit more clear.